winforms/src/toga_winforms/widgets/button.py

```diff
@@ -31,7 +33,9 @@ def set_text(self, text):
         self.native.Text = text
 
     def rehint(self):
-        # self.native.Size = Size(0, 0)
-        # print("REHINT Button", self, self.native.PreferredSize)
-        self.interface.intrinsic.width = at_least(self.native.PreferredSize.Width)
-        self.interface.intrinsic.height = self.native.PreferredSize.Height
+        self.interface.intrinsic.width = self.scale_out(
+            at_least(self.native.PreferredSize.Width), ROUND_UP
```

Comment:
Should this have a MIN_SIZE component?

Reply:
In this case I think it's better to use PreferredSize, because it reacts to the label text.